client/routes.js

```diff
@@ -3,9 +3,8 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 
 // components
-import App from './components/App';
```

Comment:
I'm not sure how things are going to work but we need this App component to wrap majority of the components because w/o it no component can access the modal component @williamolojede created.

Reply:
I will sync with @williamolojede about this.

Reply:
Yes, It has been clarified.